cbioportal_etl/scripts/genomics_file_cbio_package_build.py

```diff
@@ -229,24 +225,26 @@ def process_dgd_fusion(cbio_id_table, fusion_dir, dgd_status):
 )
 parser.add_argument(
     "-l",
-    "--legacy",
+    "--legacy", 
```

Comment:
I think after this, later on I might try and do some clean up. This "legacy" mode fusion is just not gonna be run anymore, and at worst, someone can just use a previous release